packages/artblocks/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@rabbitholegg/questdk-plugin-artblocks",
+  "version": "1.0.0-alpha.11",
```

Comment:
Please reset the version back to 1.0.0-alpha.11 (This version number is for the template package)